README.md

```diff
@@ -203,8 +203,9 @@ curl --request POST \
   --url http://localhost:4001/v1/spaces/default/subscriptions \
   --header 'content-type: application/json' \
   --data '{
+    "type": "async",
```

Comment:
I think that from UX perspective it's better to have it like that because it's more explicit what type the subscription is. It's easier to immediately understand what it actually do. Also it allows introducing more types in the future. @alexdebrie @plfx any thoughts?

In general we should discuss things like that on design phase.